content/docs/atom-guide/look-dev/tutorials.md

```diff
@@ -0,0 +1,8 @@
+---
+linktitle: Tutorials
+title: "Tutorials"
+description: "Tutorials for the Atom renderer."
+toc: true
+---
+
+To learn how to use the Atom renderer, refer to any of the [rendering tutorials in the learning guide](/docs/learning-guide/tutorials/rendering/)!
```

Comment:
As mentioned in the previous comment, consider removing this page completely.

Reply:
Removed.